.gitignore

```diff
@@ -1,2 +1 @@
-test.md
 node_modules
```

Comment:
- [ ] According to your previouse reviewer i would suggest you to add  or include test.md to you .gitignore file ,According to your project requirement 
- There is a .gitignore file listing test.md file and node_modules directory (as it can be added by accident while setting up Linters).
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.

- [ ] I can see you have no linters pass kindly setup your linters correctly follow this guildelinesSet up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.